platform/broadcom/sonic-platform-modules-ruijie/common/lib/eepromutil/onietlv.py

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import binascii
```

Comment:
i think sonic has its onie tlv parser, why do you need to introduce your own?

Reply:
We introduce our own tlv parser due to below two points:
1. Sonic onie TLV parser will print log to serial port
2. We need to add our own fields in onie TLV

Reply:
please improve general onie tlv handling.

Reply:
ok, we have planned to improve it